app/views/admin/admin/_admintab.html.erb

```diff
@@ -0,0 +1,26 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
+<!------ Include the above in your HEAD tag ---------->
+
+
+  <div class="row">
```

Comment:
even if you choose to take content from a link, mold it to your requirement before pushing out the changes.

Reply:
Have removed the file.